frontend/base/src/main/resources/GitMacheteBundle.properties

```diff
@@ -190,9 +190,9 @@ action.GitMachete.FastForwardMergeSelectedBranchToParentAction.GitMacheteContext
 action.GitMachete.FastForwardMergeSelectedBranchToParentAction.description=Fast-forward merge selected branch to parent
 
 
-action.GitMachete.MergeParentIntoSelectedBranchAction.text=Git Machete: Merge Parent Into Branch
-action.GitMachete.MergeParentIntoSelectedBranchAction.GitMacheteContextMenu.text=_Merge Parent Into Branch
-action.GitMachete.MergeParentIntoSelectedBranchAction.description=Merge parent into selected branch
+action.GitMachete.MergeParentIntoSelectedBranchAction.text=Git Machete: Checkout and Merge Parent Into Selected Branch
```

Comment:
Huh this sounds like `(Checkout&Merge) Parent`, i.e. as if it's parent that's checked out 🤔

Reply:
`Checkout Selected Branch and Merge Parent Into It` is less elegant... albeit more precise :goncern: